packages/docs-browser/src/components/DocsBrowser/StyledComponents.js

```diff
@@ -19,7 +19,7 @@ export const StyledContent = styled.div`
 
 export const StyledBreadcrumbs = styled.div`
   grid-area: breadcrumbs;
-  ${({embedded}) => embedded && `
+  ${({embedded, hasLeftPadding}) => embedded && !hasLeftPadding && `
```

Comment:
why check for embedded? I think the hasLeftPadding should not depend on the embedded property or you should document it.